### ISSUE TITLE
Transcript lags in long paragraphs.

### ISSUE THREAD

**Issue by Isac200** [hyperaudio/hyperaudio-lite-editor#231]
In instances where there is a long paragraph splitting and merging of paragraphs makes the transcript to lag, and the lag time increases gradually as the paragraph gets longer.

**Comment by maboa** [comment]
Yes, I see the issue with larger transcripts – thanks @Isac200 – I'm taking a look at that now.

**Comment by maboa** [comment]
This should be fixed now @Isac200 with this PR. Closing.
